### ISSUE TITLE
Change name of R package

### ISSUE THREAD

**Issue by StaffanBetner** [MansMeg/ada_code#1]
There is already an R packages called ada, so something else would be better.

**Comment by MansMeg** [comment]
Yes. I think I will rename it to adapop as in Ada poll of polls model.
